### ISSUE TITLE
train on my new dataset contain only one class

### ISSUE THREAD

**Issue by chenweiqian** [balancap/SSD-Tensorflow#49]
My result is very bad. The mAP is close to 0. I don't know why, hope anyone who can help me ! Thanks!!!!

**Comment by congjianting** [comment]
did you find the problems?

**Comment by chenweiqian** [comment]
@congjianting  I am very sorry that i forget it. But i know that it works normally in the end.

**Comment by congjianting** [comment]
@chenweiqian  do you have a normal project codes?   if it is, could you leave me a wechat number and send this project codes to me? thanks

**Comment by chenweiqian** [comment]
@congjianting  i have a version that changed to my requirement. if you want, i can send you through email.

**Comment by congjianting** [comment]
thanks for replying. 

do you have trained this project with your own dataset?

if you did, could the training loss value  decrease ?  The most confusing thing i met is the loss is not decreasing 

even using pascalvoc 2007.

**Comment by congjianting** [comment]
@chenweiqian thanks for replying. 

do you have trained this project with your own dataset?

if you did, could the training loss value  decrease ?  The most confusing thing i met is the loss is not decreasing 

even using pascalvoc 2007.

**Comment by chenweiqian** [comment]
@congjianting  No, I didn't use my own dataset. I'm terribly sorry that I couldn't help you.

**Comment by soEdwardzh** [comment]
@chenweiqian hello，I have the same problem to you,I train my own dataset with only one class, the network can convege ,but the mAP is low,can you give me some suggestion? thx ~

**Comment by chenweiqian** [comment]
@soEdwardzh I am sorry. It’s so long time that i forget it.

**Comment by soEdwardzh** [comment]
@chenweiqian  thx for your reply~
